quickwit-storage/src/bundle_storage.rs

```diff
@@ -327,12 +251,9 @@ mod tests {
 
         create_bundle.add_file(&test_file1_name)?;
         create_bundle.add_file(&test_file2_name)?;
-        let offsets = create_bundle.finalize()?;
-        assert!(offsets.footer_offsets.start > offsets.hotcache_offset_start);
-
-        let bytes = std::fs::read(&bundle_file_name)?;
+        create_bundle.finalize()?;
 
-        let metadata = BundleStorageFileOffsets::open(&bytes).unwrap();
+        let metadata = BundleStorageFileOffsets::open(&buffer)?;
```

Comment:
while `?` is nicer, it doesn't give good stacktraces on error. we could fix that with an error that carries the stacktraces

Reply:
Yes I agree it is a bit lame when the test fails.